src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -537,7 +537,7 @@
     <value>Learn more about administrator mode</value>
   </data>
   <data name="GeneralPage_ToggleSwitch_AutoDownloadUpdates.Header" xml:space="preserve">
-    <value>Download updates automatically</value>
+    <value>Download updates automatically (Except on metered connections, where charges may apply)</value>

```

Comment:
Technically speaking `where charges may apply` is incorrect, since a user can set a connection to be metered even if charges are not applied. For example the connection can have a per day/month limit and then simply slow down.
I suggest to just say `(Except on metered connections)`

Reply:
I will copy your comment into issue #2967. There we can discuss this.